eng/Versions.props

```diff
@@ -11,6 +11,10 @@
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
     <UsingToolNuGetRepack>true</UsingToolNuGetRepack>
     <!-- Dependencies -->
+    <MicrosoftSourceBuildIntermediatesourcebuildreferencepackagesPackageVersion>8.0.0-alpha.1.22616.1</MicrosoftSourceBuildIntermediatesourcebuildreferencepackagesPackageVersion>
+    <MicrosoftSourceLinkGitHubVersion>1.2.0-beta-22518-02</MicrosoftSourceLinkGitHubVersion>
```

Comment:
Why are these versions needed here? This repo does not require any specific version and should use the default one from Arcade SDK.

Reply:
Yes, that is why the `CoherentParentDependency` attribute is used in `Version.Details.xml`. I followed the default behaviour of having the version for each dependency specified in `Version.props` in this case

Reply:
Since they are coherent versions, I can remove them from `Versions.props` if you think that this makes sense

Reply:
Yeah, it would be better to avoid having them in Versions.props. Versions.props should only have versions that the repo needs to use that are different from the defaults that come with Arcade SDK.

Reply:
Sure thing, will remove